build.sh

```diff
@@ -287,7 +287,7 @@ if [ ! "$NO_PUSH" = 'true' ]; then
             time docker push "$GAME_IMAGE_LATEST"
         fi
     elif [ "$PIPELINE" = 'update' ]; then
-        time docker push "$GAME_IMAGE_LAYERED"
+        time docker push "$GAME_IMAGE_LATEST"
```

Comment:
#L287 already `docker push "$GAME_IMAGE_LATEST"`

Reply:
that is for `PIPELINE=build`